tests/test_transforms/test_outliers/test_outliers_transform.py

```diff
@@ -255,3 +330,103 @@ def test_params_to_tune(transform, outliers_solid_tsds):
+    ),
+)
+def test_incorrect_not_exists_column(transform, outliers_solid_tsds):
+    ts = outliers_solid_tsds
+    assert len(transform.params_to_tune()) > 0
+    with pytest.raises(ValueError, match='Name ignore_flag_column="is_holiday" not find.'):
+        transform.fit(ts)
+        _ = transform.transform(ts)
+
+
+@pytest.mark.parametrize(
+    "transform",
+    (
+        MedianOutliersTransform(in_column="target", ignore_flag_column="is_holiday"),
+        DensityOutliersTransform(in_column="target", ignore_flag_column="is_holiday"),
+        PredictionIntervalOutliersTransform(in_column="target", model="sarimax", ignore_flag_column="is_holiday"),
+    ),
+)
+def test_incorrect_type_ignore_flag(transform, outliers_solid_tsds_with_error):
+    ts = outliers_solid_tsds_with_error
+    with pytest.raises(
+        ValueError,
+        match=re.escape("Columns ignore_flag contain non binary value: columns: \"is_holiday\" in segment: ['1', '2']"),
+    ):
+        transform.fit(ts)
+        _ = transform.transform(ts)
+
+
+@pytest.mark.parametrize(
+    "transform, expected_changes",
+    [
+        (MedianOutliersTransform(in_column="target", ignore_flag_column="is_holiday"), {"change": {"target"}}),
+        (DensityOutliersTransform(in_column="target", ignore_flag_column="is_holiday"), {"change": {"target"}}),
+        (
+            PredictionIntervalOutliersTransform(in_column="target", model="sarimax", ignore_flag_column="is_holiday"),
+            {"change": {"target"}},
+        ),
+    ],
+)
+def test_full_train_with_outliner(transform, expected_changes, outliers_solid_tsds_for_pipeline):
+    ts = outliers_solid_tsds_for_pipeline
+
+    train_ts = deepcopy(ts)
+    test_ts = deepcopy(ts)
+
+    transform.fit(train_ts)
+
+    transformed_test_ts = transform.transform(deepcopy(test_ts))
+
+    inverse_transformed_test_ts = transform.inverse_transform(deepcopy(transformed_test_ts))
+
+    # check
+    assert_column_changes(ts_1=transformed_test_ts, ts_2=inverse_transformed_test_ts, expected_changes=expected_changes)
+    flat_test_df = test_ts.to_pandas(flatten=True)
+    flat_transformed_test_df = transformed_test_ts.to_pandas(flatten=True)
+    flat_inverse_transformed_test_df = inverse_transformed_test_ts.to_pandas(flatten=True)
+    created_columns, removed_columns, changed_columns = find_columns_diff(
+        flat_transformed_test_df, flat_inverse_transformed_test_df
+    )
+    pd.testing.assert_frame_equal(
+        flat_test_df[list(changed_columns)], flat_inverse_transformed_test_df[list(changed_columns)]
+    )
+
+
+@pytest.mark.parametrize(
+    "transform",
+    [
+        (MedianOutliersTransform(in_column="target", ignore_flag_column="is_holiday")),
+        (DensityOutliersTransform(in_column="target", ignore_flag_column="is_holiday")),
+        (PredictionIntervalOutliersTransform(in_column="target", model="sarimax", ignore_flag_column="is_holiday")),
+    ],
+)
+def test_full_pipeline(transform):
```

Comment:
Add holiday transform before outliers transform and use dataset without applied transforms